frontend/app-development/layout/SettingsModalButton/SettingsModal/components/Tabs/SetupTab/SetupTabContent/SetupTabContent.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { ReactNode, useState } from 'react';
+import classes from './SetupTabContent.module.css';
+import { ApplicationMetadata } from 'app-shared/types/ApplicationMetadata';
+import { useTranslation } from 'react-i18next';
+import { useAppMetadataMutation } from 'app-development/hooks/mutations';
+import { ErrorMessage, Textfield } from '@digdir/design-system-react';
+import { SwitchRow } from './SwitchRow';
+import { Divider } from 'app-shared/primitives';
+import { getIsDatesValid } from 'app-development/layout/SettingsModalButton/SettingsModal/utils/tabUtils/setupTabUtils';
+
+export type SetupTabContentProps = {
+  appMetadata: ApplicationMetadata;
+  org: string;
+  app: string;
+};
+
+/**
+ * @component
+ *    The content of the Setup Tab
+ *
+ * @property {ApplicationMetadata}[appMetadata] - The application metadata
+ * @property {string}[org] - The org
+ * @property {string}[app] - The app
+ *
+ * @returns {ReactNode} - The rendered component
+ */
+export const SetupTabContent = ({ appMetadata, org, app }: SetupTabContentProps): ReactNode => {
+  const { t } = useTranslation();
+
+  const [startDate, setStartDate] = useState<string>(appMetadata?.validFrom ?? '');
+  const [endDate, setEndDate] = useState<string>(appMetadata?.validTo ?? '');
+  const [showEndDate, setShowEndDate] = useState<boolean>(appMetadata.validTo !== undefined);
+
+  const { mutate: updateAppMetadataMutation } = useAppMetadataMutation(org, app);
+
+  const dateIsValid: boolean = getIsDatesValid(appMetadata?.validFrom, appMetadata?.validTo);
```

Comment:
Du kan vurdere om det er bedre å endre navn på denne variabelen til "isInvalidDates", siden den brukes bare til error-melding. Så slipper vi å bruke "!" tegn flere steder i koden. 😊 
```suggestion
  const isInvalidDates: boolean = !getIsDatesValid(appMetadata?.validFrom, appMetadata?.validTo);
```

Reply:
Yes 😄